moit-api/src/main/kotlin/com/mashup/moit/moit/controller/dto/MoitDto.kt

```diff
@@ -0,0 +1,71 @@
+package com.mashup.moit.moit.controller.dto
+
+import com.mashup.moit.common.util.DateTimeUtils
+import com.mashup.moit.domain.moit.Moit
+import com.mashup.moit.domain.moit.NotificationRemindLevel
+import com.mashup.moit.domain.moit.NotificationRemindOption
+import com.mashup.moit.domain.moit.ScheduleRepeatCycle
+import com.mashup.moit.domain.usermoit.UserMoit
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.DayOfWeek
+import java.time.LocalDate
+
+@Schema(description = "moit 가입 응답")
+data class MoitJoinResponse(
+    @Schema(description = "생성된 userMoitId")
+    val id: Long,
+    @Schema(description = "가입한 moitId")
+    val moitId: Long,
+    @Schema(description = "가입한 userId")
+    val userId: Long,
+    @Schema(description = "moit 내 권한", allowableValues = ["ADMIN", "MEMBER"])
+    val role: String,
+    @Schema(description = "moit 반복 요일")
+    val scheduleDayOfWeeks: Set<DayOfWeek>,
+    @Schema(description = "moit 반복 주기")
+    val scheduleRepeatCycle: ScheduleRepeatCycle,
+    @Schema(description = "moit 시작 시간", example = "HH:mm")
+    val scheduleStartTime: String,
+    @Schema(description = "moit 종료 시간", example = "HH:mm")
+    val scheduleEndTime: String,
+    @Schema(description = "moit 지각 시간")
+    val fineLateTime: Int,
+    @Schema(description = "moit 지각 벌금")
+    val fineLateAmount: Int,
+    @Schema(description = "moit 결석 시간")
+    val fineAbsenceTime: Int,
+    @Schema(description = "moit 결석 벌금")
+    val fineAbsenceAmount: Int,
+    @Schema(description = "moit 알람 리마인드 on/off")
+    val notificationIsRemindActive: Boolean,
+    @Schema(description = "moit 알람 리마인드 시간")
+    val notificationRemindOption: NotificationRemindOption?,
+    @Schema(description = "moit 알람 리마인드 매운맛")
+    val notificationRemindLevel: NotificationRemindLevel?,
+    @Schema(description = "moit 시작 일자", example = "YYYY-MM-dd")
+    val startDate: LocalDate,
+    @Schema(description = "moit 종료 일자", example = "YYYY-MM-dd")
+    val endDate: LocalDate,
+) {
+    companion object {
+        fun of(moit: Moit, userMoit: UserMoit) = MoitJoinResponse(
```

Comment:
약간 스타일 차이이긴 한데 저는 public api의 경우 
```suggestion
        fun of(moit: Moit, userMoit: UserMoit): MoitJoinResponse = MoitJoinResponse(
```
이렇게 반환 타입을 명시해주는 편인데 다른 분들은 어떠세요? ㅎㅎ